src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -219,7 +221,29 @@ private static boolean isToolTip(Node currentNode) {
         
         return false;
     }
-
+    
+    private static boolean isMotdComponent(Node currentNode) {      
+        if (currentNode.getNodeName().equalsIgnoreCase("script")) {
+            return currentNode.getTextContent().contains("closeMotd");
+            
+        } else if (currentNode.getNodeName().equalsIgnoreCase("div")) {
+            NamedNodeMap attributes = currentNode.getAttributes();
+            
+            if (attributes == null) { 
+                return false;
+            }
+                
+            for (int i = 0; i < attributes.getLength(); i++) {
+                Node attribute = attributes.item(i);
+                if (attribute.getNodeName().equalsIgnoreCase("id")
+                      && attribute.getNodeValue().contains("student-motd-container")) {
+                    return true;
+                }
+            }
+        }
```

Comment:
This for loop is going to run for every div right? Is there a faster way to determine this? Anyway, you can try it first. If it doesn't slow down tests, then no need to bother about it.